### PR TITLE
[update] Check the fields when creating or updating an experiment

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### What problem does this PR solve?
Check the fields when creating or updating an experiment

### What is changed and how does it work?
Check the fields when creating or updating an experiment

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by fewdan** [comment]
There is still a problem. . Please don't review it yet.

File: pkg/apiserver/server.go [hunk 1]
```diff
@@ -121,6 +121,7 @@ func newAPIHandlerEngine() (*gin.Engine, *gin.RouterGroup) {
 		v.RegisterValidation("DurationValid", apivalidator.DurationValid)
 		v.RegisterValidation("ValueValid", apivalidator.ValueValid)
 		v.RegisterValidation("PodsValid", apivalidator.PodsValid)
+		v.RegisterValidation("required_field_equal", apivalidator.RequiredFieldEqualValid, true)
```
Comment (cwen0) [inline pkg/apiserver/server.go:124]: Required_field_equal

File: pkg/apiserver/server.go [hunk 2]
```diff
@@ -121,7 +121,7 @@ func newAPIHandlerEngine() (*gin.Engine, *gin.RouterGroup) {
 		v.RegisterValidation("DurationValid", apivalidator.DurationValid)
 		v.RegisterValidation("ValueValid", apivalidator.ValueValid)
 		v.RegisterValidation("PodsValid", apivalidator.PodsValid)
-		v.RegisterValidation("required_field_equal", apivalidator.RequiredFieldEqualValid, true)
+		v.RegisterValidation("Required_field_equal", apivalidator.RequiredFieldEqualValid, true)
```
Comment (cwen0) [inline pkg/apiserver/server.go:124]: RequiredFieldEqual

**Pull request merged**